config/jobs/kubernetes/sig-release/kubernetes-builds.yaml

```diff
@@ -59,10 +59,41 @@ periodics:
     testgrid-alert-email: kubernetes-release-team@googlegroups.com
     description: 'Ends up running: make quick-release'
 - interval: 4h
-  name: ci-kubernetes-build-debian-unstable
+  name: ci-release-build-packages-debs
   labels:
     preset-service-account: "true"
     preset-dind-enabled: "true"
+  annotations:
+    testgrid-alert-email: kubernetes-release-managers@googlegroups.com
+    testgrid-dashboards: sig-release-master-informing
+    testgrid-tab-name: build-packages-debs
+  spec:
+    containers:
+    - image: gcr.io/k8s-testimages/bootstrap:v20190703-1f4d616
+      args:
+      - --repo=k8s.io/release
+      - --root=/go/src
+      - --timeout=180
+      - --scenario=execute
+      - --
+      - make
+      - build-debs
+      # docker-in-docker needs privileged mode
+      securityContext:
+        privileged: true
+      resources:
+        requests:
+          cpu: 4
+          memory: "8Gi"
+- interval: 4h
+  name: ci-release-build-packages-rpms
+  labels:
+    preset-service-account: "true"
+    preset-dind-enabled: "true"
+  annotations:
```

Comment:
The job already has an `annotations` key further down, thats why the yamllint fails.